File: 240116/두 사람/two-person.py
arr1 = input().split()
arr2 = input().split()

age1 = int(arr1[0])
gender1 = arr1[1]

age2 = int(arr2[0])
gender2 = arr2[1]

if (age1 >= 19 and gender1 == "M") or (age2 >= 19 and gender2 == "M"):
    print(1)
else:
    print(0)

File: 240409/두 사람/two-person.swift
let info1 = readLine()!.split(separator: " ").map {$0}
let info2 = readLine()!.split(separator: " ").map {$0}
let (age1, gender1) = (Int(info1[0]), info1[1])
let (age2, gender2) = (Int(info2[0]), info2[1])

var answer = 0

if age1! >= 19 || age2! >= 19 {
    if gender1 == "M" || gender2 == "M" {
        if age1! >= 19 && gender1 == "M" {
            answer = 1
        }

        if age2! >= 19 && gender2 == "M" {
            answer = 1
        }
    }
} 

print(answer)